### DISCUSSION TITLE
Phase distribution

### DISCUSSION THREAD

**Discussion by my-name-iz-ayush** [neperfepx/neper#719] [Q&A] [answered]
Hello @rquey . I'm tessellating a 2D microstructure containing 40% volume fraction of phase 1 and remaining phase 2 using following code:
```
neper -T -n 50 -morpho "diameq:normal(0.030,0.001),1-circularity:normal(0.010,0.003)" -dim 2 -group "id<=20?1:2" -o ti64
```
The generated microstructure kind of contains agglomerated phase 1. how can I evenly distribute it among the microstructure? and if any suggestion for assigning group using volume fraction in 2D microstructure?
![free_group](https://github.com/neperfepx/neper/assets/78230636/592dc6cc-adb5-4cc9-bd2a-7d564f143bae)

**Comment by rquey** [comment, marked as answer]
There is no metric to control how much cells agglomerate, but this could probably be added.
